Frameworks/UIKit/StarboardXaml/LayerAnimation.cpp

```diff
@@ -143,12 +143,14 @@ concurrency::task<Layer^> _SnapshotLayer(Layer^ layer) {
 
             // Return a new 'copy' layer with the rendered content
             Layer^ newLayer = ref new Layer();
+            newLayer->Name = "_Temporary_CoreAnimation_SnapshotLayer";
             LayerCoordinator::InitializeFrameworkElement(newLayer);
 
             // Copy display properties from the old layer to the new layer
             LayerCoordinator::SetValue(newLayer, "opacity", LayerCoordinator::GetValue(layer, "opacity"));
             LayerCoordinator::SetValue(newLayer, "position", LayerCoordinator::GetValue(layer, "position"));
-            LayerCoordinator::SetValue(newLayer, "size", LayerCoordinator::GetValue(layer, "size"));
```

Comment:
switched back to original implementation here

---
In reply to: [87917684](https://github.com/Microsoft/WinObjC/pull/1378#discussion_r87917684) [](ancestors = 87917684)